install.sh

```diff
@@ -33,11 +34,6 @@ set -o errexit
 set -o errtrace
 set -o pipefail
 
-if ! [ $(id -u) = 0 ]; then 
```

Comment:
Move this after the arguments processing in order to allow running `./install.sh --help` without requiring sudo.